docs/gettingstarted/installing.md

```diff
@@ -88,31 +88,31 @@ alienv enter O2Physics/latest ninja/latest
 Go to the build directory
 
 ```bash
-cd sw/BUILD/O2Physics-latest/O2Physics
+cd ~/alice/sw/BUILD/O2Physics-latest/O2Physics
 ```
 
 You can now rebuild a specific directory with
 
 ```bash
-ninja install <your-analysis-directory>/all
+ninja <your-analysis-directory>/install
 ```
 
 For example:
 
 ```bash
-ninja install PWGCF/Tasks/all
+ninja PWGCF/Tasks/install
 ```
 
 A specific executable can be built with
 
 ```bash
-ninja install O2Physicsexe-<target>
+ninja O2Physicsexe-<target>
```

Comment:
So here it will not be installed anymore?

Reply:
Then we should also explain this. I think from the current instructions, people would expect that the binary is also in the path. So we have to explain here in addition, how to execute it in the stage directory.

Reply:
Good point. @vkucera could you please add a sentence about that?

Reply:
I think the best would be to have a target which installs the executable called like the executable itself. it should be doable, at least with ninja

Reply:
@aalkin @jgrosseo Updated